contracts/dollar/hardhat.config.ts

```diff
@@ -52,43 +52,10 @@ const config: HardhatUserConfig = {
     timeout: 1000000,
   },
   namedAccounts: {
-    ubq: {
-      default: "0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd", //  without UBQ => impersonate
-      mainnet: 0, // use default account 0 with UBQ (of same address !) on mainnet
-    },
-    whaleAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-    tester: "0x89eae71b865a2a39cba62060ab1b40bbffae5b0d",
-    sablier: "0xA4fc358455Febe425536fd1878bE67FfDBDEC59a",
-    DAI: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
-    USDC: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
-    USDT: "0xdAC17F958D2ee523a2206206994597C13D831ec7",
-    curve3CrvBasePool: "0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7",
-    curve3CrvToken: "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490",
-    curveFactory: "0x0959158b6040D32d04c301A72CBFD6b39E21c9AE",
-    curveWhaleAddress: "0xC2872ab688400940d5a6041599b3F7de20730d49",
-    daiWhaleAddress: "0x16463c0fdB6BA9618909F5b120ea1581618C1b9E",
-    sushiMultiSig: "0x9a8541Ddf3a932a9A922B607e9CF7301f1d47bD1",
-    UbqWhaleAddress: "0xa53A6fE2d8Ad977aD926C485343Ba39f32D3A3F6",
-    MasterChefAddress: "0x8fFCf9899738e4633A721904609ffCa0a2C44f3D",
-    MetaPoolAddress: "0x20955cb69ae1515962177d164dfc9522feef567e",
-    BondingAddress: "0x831e3674Abc73d7A3e9d8a9400AF2301c32cEF0C",
-    BondingV2Address: "0xdAC17F958D2ee523a2206206994597C13D831ec7", // FAKE ADDRESS, TO BE REPLACED AFTER V2 DEPLOYMENT
-    UbiquityAlgorithmicDollarManagerAddress: "0x4DA97a8b831C345dBe6d16FF7432DF2b7b776d98",
-    jarUSDCAddr: "0xEB801AB73E9A2A482aA48CaCA13B1954028F4c94",
-    jarYCRVLUSDaddr: "0x4fFe73Cf2EEf5E8C8E0E10160bCe440a029166D2",
-    strategyYearnUsdcV2: "0xEecEE2637c7328300846622c802B2a29e65f3919",
-    usdcWhaleAddress: "0x72A53cDBBcc1b9efa39c834A540550e23463AAcB",
-    pickleControllerAddr: "0x6847259b2B3A4c17e7c43C54409810aF48bA5210",
+    deployer: { default: 0 },
+    alice: { default: 1 },
+    bob: { default: 2 }
```

Comment:
so it was using constant values for deployed contracts on mainnet. 
It didn't make sense to me and seems like hardcoded. What I was thinking here is just that we should be able to handle the constant variables properly per chains. so I moved them tasks/constant/index.ts

Here, deployer, alice & bob are the names for the accounts you put using mnemonics. 
If this is totally out of your origin codebase, I can get them restored. -- For some of the other codes, I can get them fixed as soon as possible if you agree to keep this.

Reply:
if you're curious to have test accounts, we have a description of some of the generic hardhat test accounts in the README that are automatically imported for the faucet script and stuff. something to consider

Reply:
Ok. I restored namedAccounts. so I keep both namedAccounts and constants. but it doesn't make 100% sense because the addresses in the namedAccounts are the mainnet addresses which is limited to mainnet. so that would be better to have them organized later